### DISCUSSION TITLE
keine Template- und Icon-Auswahl im neuen Log-Layout

### DISCUSSION THREAD

**Discussion by Donnerknispel** [2Abendsegler/GClh#1865] [Questions] [answered]
Heute bin ich irgendwie im neuen Log-Layout gelandet. Keine Ahnung wie lange es dies schon gibt. Dabei ist mir aufgefallen, dass es hier keine Template- und Icon-Auswahl, wie im alten Layout, gibt, Zumindest bei mir habe ich keine gesehen. Muss ich irgendwo etwas konfigurieren? Gibt es das aktuell nicht im neuen Layout? Habe ich etwas übersehen und es gibt das schon?
Ich habe die Version 0.11.5.1 installiert.

Gruß Donnerknispel

![Alt](https://user-images.githubusercontent.com/7745027/128605340-d09a48c3-f5cf-4476-afb7-c1458d8fa0ea.jpg)
![Neu](https://user-images.githubusercontent.com/7745027/128605345-c1899811-c0ab-4e08-9858-f8e648fab76b.jpg)

**Comment by capoaira** [comment, marked as answer]
Moin,
Danke für das Melden des Fehlers.
Das neue Design gibt es schon länger. Leider gab es vor ein paar Tagen/Wochen ein Update durch das unser Code nicht mehr funktioniert. 
Wir haben schon ein Issue dazu: #1855
Dort ist auch beschrieben, wie es zum alten Layout zurück geht.

**Comment by Donnerknispel** [comment]
Danke für die fixe Reaktion.

Ohne den Helper wäre das Leben viel schwieriger.

Erst wenn was nicht funktioniert, merkt man wie unhandlich das „Original“ ist.

 

Danke

Olaf
